frontend/docs/ROADMAP.md

```
@@ -0,0 +1,377 @@
+  npm install -D @playwright/test
+  npx playwright install
+  ```
+
+- [ ] **테스트 시나리오 작성**
+  - Control 그룹 사용자 플로우
+  - Treatment 그룹 사용자 플로우
+  - 에러 상황 처리
+  - 설문 제출 플로우
+
+- [ ] **테스트 파일 생성**
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)